src/mixins/input.js

```diff
@@ -79,6 +79,11 @@ export default {
     __onClick (e) {
       this.focus()
       this.$emit('click', e)
+    },
+    __onPaste (e) {
+      this.focus()
```

Comment:
Why is the focus() necessary? I assume the field is already focused when user pastes in content.